src/msg/async/AsyncConnection.cc

```diff
@@ -82,7 +82,7 @@ class C_clean_handler : public EventCallback {
  public:
   explicit C_clean_handler(AsyncConnectionRef c): conn(c) {}
   void do_request(int id) {
-    conn->cleanup_handler();
+    conn->cleanup();
```

Comment:
> It needs all fd used locally which not like kernel socket shared by all cores.

so you mean, we need to pin the fds to its own EventCenter to prepare for the DPDK?  but how is this mapped to DPDK?